adagrams/game.py

```diff
@@ -1,11 +1,151 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    #Create a set of 10 letters that matches the letter frequency distribution#
+    draw = []
+    letter_pool_copy = LETTER_POOL.copy()
+    letters_list = list(letter_pool_copy.keys())
+
+    while len(draw) < 10:       
+        #Take a random number within the len of letter pool and get the letter in that index
+        random_letter = letters_list[random.randint(0,len(letter_pool_copy)-1)]
+        
+        #Check if the letter is still availabe in the letter pool and add it to the hand
+        if letter_pool_copy[random_letter] >= 1:
```

Comment:
A more pythonic way of writing this line would be:
```python
if letter_pool_copy[random_letter]:
```
It works the same as what you have since if ` letter_pool_copy[random_letter]` is `0` then it will be a falsy value.

Reply:
Noted! 📝